pkg/syncer/metadatasyncer.go

```diff
@@ -330,6 +342,26 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 				return logger.LogNewErrorf(log, "failed to start informer on %q instances. Error: %v",
 					cnsoperatorv1alpha1.CnsStoragePolicyQuotaSingular, err)
 			}
+
+			go func() {
```

Comment:
why a go routine? anyways `startAvailabilityZoneCRInformer` starts another one. Let's follow the same format as `startStoragePolicyQuotaCRInformer` and `initCnsVolumeOperationRequestCRInformer`

Reply:
We will be waiting till all CnsVolumeInfo CRs are patched with "zones" info, hence we need a go routine.